matchmaker/views/external_api_2_3_tests.py

```diff
@@ -73,11 +75,17 @@ def test_mme_match_proxy(self, mock_post_to_slack, mock_email):
 
         response = self._make_mme_request(url, 'post')
         self.assertEqual(response.status_code, 400)
-        self.assertDictEqual(response.json(), {'message': 'No JSON object could be decoded'})
+        response_json = response.json()
+        self.assertListEqual(list(response_json.keys()), ['message'])
+        self.assertIn(response_json['message'], ['No JSON object could be decoded',
+                                                 'Expecting value: line 1 column 1 (char 0)'])
```

Comment:
The error message in python 3 is a lot less readable than in python 2 here. Could you update the api's error handling to explicitly return an error message here (similar to what you did with the redis utils)

Reply:
It is a little different from the Redis utils, but I will change it to a single unified error message.